### ISSUE TITLE
[nsfw] xhamster19.desi: ads

### ISSUE THREAD

**Issue by Malgotra** [uBlockOrigin/uAssets#17200]
### Prerequisites

- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I forced an update of my filter lists. (Click the ["Purge all caches"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches) button while holding the 'Shift' key, then click the ["Update now"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now) button.)
- [X] I did not remove any of the [default filter lists](https://user-images.githubusercontent.com/585534/91296476-ab9d8c00-e76a-11ea-9fb9-87d6861bdf73.png), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable external filter lists, or I have verified that the issue still occurs without enabling external filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin (uBO) along with other content blocker extensions.
- [X] I have verified that the web browser's built-in blocker or [DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking) (standalone or through a VPN) is not causing the issue.
- [X] I did not answer truthfully to **ALL** the above checkpoints.

### URL address of the web page

`https://xhamster19.desi/`

### Category

ads

### Description

Ads leftover

### Screenshot(s)

![Screenshot_20230316-184508](https://user-images.githubusercontent.com/122884037/225628671-db68b5af-eba3-4263-ac10-02b97a58ffe6.png)
![Screenshot_20230316-184519](https://user-images.githubusercontent.com/122884037/225628689-214f6480-c5af-4d26-9e76-4ae56c0ae7f5.png)
![Screenshot_20230316-184536](https://user-images.githubusercontent.com/122884037/225628706-74eed344-dbfe-4a93-b4b3-bea01c7ff534.png)

### Configuration

<details>

```yaml
uBlock Origin: 1.47.4
Firefox Mobile: 110
filterset (summary): 
  network: 184610
  cosmetic: 223007
  scriptlet: 38525
  html: 1101
listset (total-discarded, last-updated): 
  added: 
    adguard-generic: 68631-5242, 8m
    adguard-spyware-url: 941-4, 8m
    block-lan: 44-0, 8m
    curben-phishing: 55653-25, 8m
    curben-pup: 191-0, 8m
    adguard-annoyance: 65057-5025, 8m
    adguard-social: 19992-35, 8m
    fanboy-thirdparty_social: 68-0, 8m
    fanboy-annoyance: 81990-70260, 8m
    fanboy-cookiemonster: 46133-247, 8m
    fanboy-social: 22245-934, 8m
    ublock-annoyances: 4894-4, 8m
    dpollock-0: 11342-502, 8m
  default: 
    user-filters: 0-0, never
    ublock-filters: 33990-100, 7m
    ublock-badware: 4720-0, 7m
    ublock-privacy: 341-0, 7m
    ublock-abuse: 78-0, 7m
    ublock-unbreak: 1944-1, 7m
    ublock-quick-fixes: 383-0, 7m
    adguard-mobile: 8553-98, 9m
    easylist: 64026-797, 7m
    easyprivacy: 31054-1866, 7m
    urlhaus-1: 6710-0, 7m
    plowe-0: 3757-2, 7m
filterset (user): [empty]
modifiedUserSettings: 
  ignoreGenericCosmeticFilters: false
  showIconBadge: false
  webrtcIPAddressHidden: true
modifiedHiddenSettings: [none]
supportStats: 
  allReadyAfter: 3407 ms
  maxAssetCacheWait: 2145 ms
popupPanel: 
  blocked: 13
  blockedDetails: 
    xhamster19.desi: 7
    google.com: 3
    googletagmanager.com: 1
    tsyndicate.com: 1
    xhcdn.com: 1
```
</details>

**Comment by Malgotra** [comment]
@mapx- ads leftovers are still visible after filter update.

**Comment by mapx-** [comment]
Do you see in the logger

`##.ytd-j`

?

**Comment by Malgotra** [comment]
Is this is logger?
![Screenshot_20230318-174815](https://user-images.githubusercontent.com/122884037/226106163-cba4ca6a-a97e-4706-94c9-b69c24bba7cb.png)
![Screenshot_20230318-174826](https://user-images.githubusercontent.com/122884037/226106167-3c0ca932-6a61-4b8c-8917-28efd8c602f4.png)

Ads leftovers are still visible.
![Screenshot_20230318-174912](https://user-images.githubusercontent.com/122884037/226106252-5f6d2cba-bf54-4ff4-9734-8fcf37c5c993.png)
![Screenshot_20230318-174852](https://user-images.githubusercontent.com/122884037/226106248-e86523c5-8ec5-4e48-aae4-7d0150ea2fd8.png)

**Comment by Malgotra** [comment]
I opened the logger and search for ##ytd-j nothing found.

**Comment by Malgotra** [comment]
![Screenshot_20230318-191526](https://user-images.githubusercontent.com/122884037/226109920-01894a76-7100-4339-aa26-07e3e95fd774.png)

**Comment by mapx-** [comment]
Did you update all the filter lists ?
Are you sure "ignore Generic Cosmetic Filters" is disabled ?

**Comment by Malgotra** [comment]
Yes  "ignore Generic Cosmetic Filters" is disabled .I just checked ##.yxd-j remove ads leftover.

**Comment by mapx-** [comment]
so, in your case, it's `##.yxd-j` and not `##.ytd-j` ?

**Comment by garry-ut99** [comment]
>  * [x]  I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)

You didn't, you created a duplicate of your own previous issue: https://github.com/uBlockOrigin/uAssets/issues/17022

>  * [x]  I did not answer truthfully to **ALL** the above checkpoints.

True, in the both  issues.

> ads leftovers are still visible after filter update.

True, I can reproduce, need `##.yxd-j`, EasyList aded both `t` and `x` as well: [LINK](https://github.com/easylist/easylist/blob/master/easylist_adult/adult_specific_hide.txt#L708-L713)
